submissions/LuckyDnepr/Hooli-popup-task/styles-menu.css

```diff
@@ -0,0 +1,248 @@
+html, body {
+    width: 90%;
+    height: 100%;
+    margin: 0px;
+}
+
+html {
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+}
+
+a {
+    text-decoration: none;
+    font-size: 16px;
+    color: black;
+}
+
+/*Menu button styles start*/
+
+#nav_keeper_input {
+    position: absolute;
+    display: block;
+    height: 50px;
+    width: 55px;
+    top: 12px;
+    left: 15px;
+    z-index: 5;
+    opacity: 0;
+    cursor: pointer;
+    padding: 5px;
+}
+
+.nav_keeper_lines {
+  height: 26px;
+  width: 32px;
+  position: absolute;
+  top: 17px;
+  left: 20px;
+  z-index: 2;
+  padding: 10px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.nav_keeper_line {
+    display: block;
+    height: 4px;
+    width: 100%;
+    border-radius: 10px;
+    background: #000000;
+}
+
+.nav_keeper_lines .line1 {
+    transform-origin: 0% 0%;
+    transition: transform 0.4s ease-in-out;
+  }
+  
+.nav_keeper_lines .line2 {
+    transition: transform 0.2s ease-in-out;
+  }
+  
+.nav_keeper_lines .line3 {
+    transform-origin: 0% 100%;
+    transition: transform 0.4s ease-in-out;
+  }
+
+#nav_keeper_input:focus + .nav_keeper_lines {
+    border:1px solid rgb(250, 128, 114);
+    border-radius: 5px;
+    box-shadow: 5px 5px 5px grey;
+}
+
+/*Menu button styles end*/
+  
+
+.nav {
+    display: flex;
+    flex-direction: column-reverse;
+    width: 25%;
+    position: absolute;
+    left: 0px;
+    top: 70px;
+    transform: translate(-150%);
+    transition: transform 0.2s ease-in-out;
+}
+
+.menu_icons {
+    width: auto;
+    height: 20px;
+    padding: 10px;
+}
+
+.nav_menu {
+    padding: 10px;
+    margin: 0;
+    width: 90%;
+    height: 250px;
+    overflow-y:hidden;
+    align-self:center;
+    gap: 5px;
+    display: grid;
+    grid-template-rows: repeat(4, 3.5em);
+    grid-template-columns: 45% 45%;
+    justify-items: stretch;
+    justify-content: center;
+
+
+}
+
+.nav_menu::-webkit-scrollbar {
+    width: 0px;
+}
+
+.nav_menu_item {
+    cursor: pointer;
+    list-style-type: none;
+    border: 1px solid rgb(250, 128, 114);
+    box-shadow: 5px 5px 5px rgba(250, 128, 114, 0.5);
+    border-radius: 5px;
+    align-self: stretch;
+    margin: 2px 0;
+    background-image: linear-gradient(white 90%, salmon 100%);
+    transition: text-shadow linear 200ms;
+    
+    
+}
+
+.show_more_item {
+    display: none;
+}
+
+.nav_menu_item:hover {
+    text-shadow: 0px -3px 8px salmon;
+    background-image: linear-gradient(white 75%, salmon 100%);
+}
+
+.nav_item_marker {
+    opacity: 0;
+}
+
+.nav_menu_item_link:focus>.nav_menu_item_link_container>.nav_item_marker {
+    opacity: 100%;
+}
+
+.nav_menu_item_link:focus>.nav_menu_item_link_container>p {
```

Comment:
try not to use html tag selectors ( `p`, `span`, `div`, ... ) for styling. You can apply this style for all `.nav_menu_item_link_container` or add class for `p`.

[Please read ](https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Cascade_and_inheritance)